docs/getting-started/users-page.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[users-page]]
+= Users
+
+The Users page provides a comprehensive overview of metric user data to understand authentication and usage information within your environment. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information you can view:
+
+[role="screenshot"]
+image::images/users/users-page.png[]
+
+*User Key Performance Indicators (KPIs) charts*
+
+KPI charts show the total number of users, successful and failed user authentications, and unique IPs based on the time range specified in the date picker. Data in the KPI charts is visualized through linear and bar graphs.
```

Comment:
We don't display `unique IPs` KPI on the user page. It is displayed on the Host page.